pmd-core/src/main/java/net/sourceforge/pmd/util/log/ConsoleLogHandler.java

```diff
@@ -14,7 +14,9 @@
  * Log to the console using a basic formatter.
  *
  * @author Wouter Zelle
+ * @deprecated This class will be complety removed in 7.0.0
  */
+@Deprecated
 public class ConsoleLogHandler extends Handler {
```

Comment:
So, the culprit was actually this class "ConsoleLogHandler", correct? Because it simply wrote to stdout...

Reply:
Correct!